OMCompiler/Compiler/BackEnd/Tearing.mo

```diff
@@ -1697,29 +1697,29 @@ try
   (_,aMatrix,aMatrixT,_,_) := BackendDAEUtil.getIncidenceMatrixScalar(isyst,BackendDAE.SOLVABLE(), SOME(ishared.functionTree));
 
 
-   try    // ToDo: Match me
-    {} := discreteVars;
-   else
+    _:= match(discreteVars)
```

Comment:
I'm not shure about it but I would guess a simpe if-statement would be easier than a match withour output. You only check if `discreteVars` is empty or not.